tests/snuba/api/endpoints/test_organization_events_histogram.py

```diff
@@ -331,6 +346,31 @@ def test_histogram_simple_using_min_max(self):
             ]
             assert response.data == self.as_response_data(expected), f"failing for {array_column}"
 
+    def test_histogram_simple_using_min_out_of_range_of_implicit_max(self):
+        # range is [0, 5), so it is divided into 5 buckets of width 1
```

Comment:
This comment is out of date here. Should say something like

> All these events are out of range, and should not appear in the results.

Reply:
I've renamed the comment.